lib/api/owner.ts

```diff
@@ -39,7 +40,7 @@ export interface OwnerReq {
 }
 
 export async function postSignup(signup: PostSignup): Promise<OwnerRes> {
-  const frontend = { url: process.env.NEXT_PUBLIC_FRONTEND_URL }
+  const frontend = isCareEnv ? 'care.rcvr.app' : 'rcvr.app'
```

Comment:
From slack:

```
Warum nicht NEXT_PUBLIC_FRONTEND_URL nur lokal machen, und wenn die nicht da ist (auf prod) dann wird per default rcvr.app oder care.rcvr.app genutzt, je nach build variant

const FRONTEND_URLS = {
  rcvr: 'https://rcvr.app',
  care: 'https://care.rcvr.app',
}
const frontendUrl = process.env.NEXT_PUBLIC_OVERWRITE_FRONTEND_URL || FRONTEND_URLS[BUILD_ENV]
```

Reply:
It still misses the part where we want to use process.env.NEXT_PUBLIC_FRONTEND_URL if it is set, and we probably want to remove that variable also from the env then.